website/public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
Add in a favicon. Feel free to create one or use the one we have for OSC's website.

Reply:
Using one from osc website